hail/python/test/hail/methods/test_impex.py

```diff
@@ -289,6 +289,22 @@ def test_import_vcfs_subset(self):
         self.assertTrue(vcf2._same(filter1))
         self.assertEqual(len(parts), vcf2.n_partitions())
 
+    def test_vcf_parser_golden_master(self):
+        # the three matrix tables referenced here were generated using the old VCF parser
+        # parser
```

Comment:
delete

Reply:
ok, I still feel that knowing how these files were generated if this test ever fails will be useful.